tests/gtest/datahandle_attribute_test.cpp

```diff
@@ -233,4 +277,59 @@ TEST_F(DatahandleAttributeTest, Attribute_Different_Size_Double) {
     delete subvolume;
 }
 
+TEST_F(DatahandleAttributeTest, Attribute_Double_Data_Outside_Intersection_Horizontal_Plane) {
```

Comment:
I think this test needs a few comments (and perhaps a little sketch) to understand what is going on.